clients/include/rocblas_common.yaml

```diff
@@ -173,6 +175,19 @@ Single double joined: &single_double_complex_real_in_complex_out
   - *single_precision_complex_real_in_complex_out
   - *double_precision_complex_real_in_complex_out
 
+#############################################
+#           Used for Dot (quick)            #
+#############################################
+Half bfloat single double complex real: &half_bfloat_single_double_complex_real_precisions
+  - *half_precision
+  - *bfa_precision
```

Comment:
I think bf16 is more descriptive than bfa. Is it possible to make this bf16?

Reply:
Good point, changed in #f746ec6.